src/appshell/view/dockwindow/eventswatcher.h

```diff
@@ -1,21 +1,24 @@
-//=============================================================================
-//  MuseScore
-//  Music Composition & Notation
-//
-//  Copyright (C) 2020 MuseScore BVBA and others
-//
-//  This program is free software; you can redistribute it and/or modify
-//  it under the terms of the GNU General Public License version 2.
-//
-//  This program is distributed in the hope that it will be useful,
-//  but WITHOUT ANY WARRANTY; without even the implied warranty of
-//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-//  GNU General Public License for more details.
-//
-//  You should have received a copy of the GNU General Public License
-//  along with this program; if not, write to the Free Software
-//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
-//=============================================================================
+/**
+ * // SPDX-License-Identifier: GPL-3.0-only
+ * // MuseScore-CLA-applies
+ * //=============================================================================
+ * //  MuseScore
+ * //  Music Composition & Notation
+ * //
+ * //  Copyright (C) 2021 MuseScore BVBA and others
+ * //
```

Comment:
a little redundant, I would like to see just 
```
/**

*/
```
or 
```
//
//
```

Reply:
or better just `/*` not `/**`
Because it is better for QtCreator to highlight it as a simple comment - dull gray, and not as a doxygen comment - bright green (for me, but I think so for others :))